app/helpers/application_helper.rb

```diff
@@ -53,4 +53,49 @@ def render_index_grid(grid)
     no_records = grid.current_page_records.length < 1
     render "admin/common/grid", no_records: no_records, grid: grid
   end
+
```

Comment:
Good point, probably not worth spending too much time on it, since it will be going away.